robotoff/settings.py

```diff
@@ -61,6 +61,9 @@ def get(self):
 JSONL_MIN_DATASET_PATH = DATASET_DIR / "products-min.jsonl.gz"
 DATASET_CHECK_MIN_PRODUCT_COUNT = 1000000
 
+
+# Not sure if it's the OCR JSONL or the products JSONL
```

Comment:
Downloaded the file and read first 10 lines - it's the product JSONL, can you update the comment?